.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  - init: go get -v -t -d ./...
+    command: go run cmd/venom/main.go -h
```

Comment:
If the goal of this integration is to have dev environment for dev, i suggest to try to use the Makefile to build and maybe run the tests ?

Reply:
The goal is to have an IDE in the cloud/in the browser. But not only an IDE but a complete dev environment.
No need to install, in your local machine(s), the correct version of Go, all the tools, the extensions in VSCode... it's one of the power of GitPod.

Reply:
Great ! 
So I guess we should try to only relies on the Makefile
We don't need the `go get` and `make build` should be fine

Reply:
When I am developing an application in Go, I don't want to build my binary but to make a go run in order to test in live my changes, for example.
I just want to have my go dependencies in order to then execute `go run` command.